api/miner_test.go

```diff
@@ -35,10 +35,12 @@ func TestIntegrationMinerGET(t *testing.T) {
 	if mg.StaleBlocksMined != staleBlocksMined {
 		t.Error("stale blocks mined is incorrect")
 	}
-	if mg.CPUHashrate != st.server.miner.CPUHashrate() {
+	CPUHashrate := st.server.miner.CPUHashrate()
+	if mg.CPUHashrate != CPUHashrate {
```

Comment:
Oops, this is a holdover from when I was making miner.CPUMining() return an error.  Will return to original in next commit.